pkg-manager/resolve-dependencies/src/index.ts

```diff
@@ -281,6 +280,21 @@ export async function resolveDependencies (
     }
   }))
 
+  let updatedCatalogs: Record<string, Record<string, string>> | undefined
+  for (const project of projectsToResolve) {
+    if (!project.updatePackageManifest) continue
+    const resolvedImporter = resolvedImporters[project.id]
+    for (let i = 0; i < resolvedImporter.directDependencies.length; i++) {
+      if (project.wantedDependencies[i]?.updateSpec == null) continue
+      const dep = resolvedImporter.directDependencies[i]
+      if (dep.catalogLookup) {
+        updatedCatalogs ??= {}
+        updatedCatalogs[dep.catalogLookup.catalogName] ??= {}
+        updatedCatalogs[dep.catalogLookup.catalogName][dep.alias] = dep.normalizedBareSpecifier ?? dep.catalogLookup.userSpecifiedBareSpecifier
+      }
```

Comment:
We already have `continue` above, so `continue` here would be more symmetrical (and less indent).

```suggestion
      if (!dep.catalogLookup) continue
      updatedCatalogs ??= {}
      updatedCatalogs[dep.catalogLookup.catalogName] ??= {}
      updatedCatalogs[dep.catalogLookup.catalogName][dep.alias] = dep.normalizedBareSpecifier ?? dep.catalogLookup.userSpecifiedBareSpecifier
```

Anyway, it's just a small nitpick. Feel free to ignore.

Reply:
I like this suggestion!